fastlane/Fastfile

```diff
@@ -469,30 +469,52 @@ import "./ScreenshotFastfile"
   # This lane builds the app with pending translations for reviewers
   # -----------------------------------------------------------------------------------
   # Usage:
-  # bundle exec fastlane build_for_translation_review
+  # bundle exec fastlane build_for_translation_review version_name:1234
   #
   # Example:
-  # bundle exec fastlane build_for_translation_review
+  # bundle exec fastlane build_for_translation_review version_name:1234
   #####################################################################################
   desc "Builds and uploads for translation review"
   lane :build_for_translation_review do | options |
     ios_build_preflight() unless (options[:skip_prechecks])
 
-    sh("cd ../.. && ./Scripts/update-translations.rb review current")
-    sh("cd ../.. && ./Scripts/update-translations.rb review waiting")
-    sh("cd ../.. && ./Scripts/update-translations.rb review fuzzy")
+    puts "Building version: #{options[:version_name]}"
 
-    ios_merge_translators_strings(strings_folder:"../WordPress/Resources/")
-    sh("cd ../.. && ./Scripts/extract-framework-translations.swift")
+    update_translations_script_path = File.join( PROJECT_ROOT_FOLDER, 'Scripts', 'update-translations.rb' )
+    sh("#{update_translations_script_path} review current")
+    sh("#{update_translations_script_path} review waiting")
+    sh("#{update_translations_script_path} review fuzzy")
```

Comment:
Same remark as with `system` about spaces-in-path + shell parsing skip performance + injection/security vulnerability:

```suggestion
    sh(update_translations_script_path, "review", "current")
    sh(update_translations_script_path, "review", "waiting")
    sh(update_translations_script_path, "review", "fuzzy")
```

Reply:
Love this – addressed in 868f344b5083164afb48d5460be0c660cfec23cb to handle other instances as well (in two cases, I ditched the use of `sh` entirely since it was trivial to do the work in Ruby instead)